src/hotspot/cpu/x86/c1_CodeStubs_x86.cpp

```diff
@@ -310,12 +310,12 @@ void NewTypeArrayStub::emit_code(LIR_Assembler* ce) {
 // Implementation of NewObjectArrayStub
 
 NewObjectArrayStub::NewObjectArrayStub(LIR_Opr klass_reg, LIR_Opr length, LIR_Opr result,
-                                       CodeEmitInfo* info, bool is_inline_type) {
+                                       CodeEmitInfo* info, bool is_null_free) {
```

Comment:
Would you mind making the same change in `c1_CodeStubs_aarch64.cpp` to keep them in sync?

Reply:
Sure, sorry for the breakage of the aarch64 platform.

Fred